R/add_sda_target.R

```diff
@@ -1,7 +1,7 @@
 #' Add portfolio level SDA emissions factor targets
 #'
-#' This function calculates the portfolio-level emissions factor targets using
-#' the sectoral-decarbonization approach.
+#' This function calculates emissions-factor targets at the portfolio-level. It
+#' uses the sectoral-decarbonization approach.
```

Comment:
Do we have a link with details about what is the sectoral-decarbonization approach?

Reply:
I think the vignette/ article I wrote is the best bet